src/main/java/spoon/support/compiler/jdt/PositionBuilder.java

```diff
@@ -420,6 +420,13 @@ SourcePosition buildPositionCtElement(CtElement e, ASTNode node) {
 			AllocationExpression allocationExpression = (AllocationExpression) node;
 			if (allocationExpression.enumConstant != null) {
 				FieldDeclaration fieldDeclaration = allocationExpression.enumConstant;
+
+				Annotation[] annotations = allocationExpression.enumConstant.annotations;
```

Comment:
There are comments below, starting with `1)`. I think it would make sense to add one for annotations too and change the numeration accordingly.

Reply:
Agreed, I have updated the comments as proposed